qt/widgets/common/inc/MantidQtWidgets/Common/DropEventHelper.h

```diff
@@ -0,0 +1,23 @@
+#ifndef MANTIDQT_MANTIDWIDGETS_DRAGEVENTHELPER_H
+#define MANTIDQT_MANTIDWIDGETS_DRAGEVENTHELPER_H
+
+#include "MantidQtWidgets/Common/DllOption.h"
+
+#include <QDropEvent>
+#include <vector>
+
+namespace MantidQt {
+namespace MantidWidgets {
+
+class EXPORT_OPT_MANTIDQT_COMMON DropEventHelper {
```

Comment:
Any point in having a class here? Couldn't this just be done with free functions and a new namespace?